src/logging/WriterFrontend.h

```diff
@@ -90,7 +154,30 @@ class WriterFrontend {
      *
      * This method must only be called from the main thread.
      */
-    void Write(int num_fields, threading::Value** vals);
+    [[deprecated("Remove in v8.1. Usedetail::LogRecord instead.")]] void Write(int num_fields, threading::Value** vals);
```

Comment:
```suggestion
    [[deprecated("Remove in v8.1. Use detail::LogRecord instead.")]] void Write(int num_fields, threading::Value** vals);
```

Reply:
Do we need this deprecation, or can the method just go?

Reply:
Leftover. thanks!